assets/panel/debugger.properties

```diff
@@ -567,6 +567,7 @@ expressions.remove.tooltip=Remove watch expression
 xhrBreakpoints.header=XHR Breakpoints
 xhrBreakpoints.placeholder=Break when URL contains
 xhrBreakpoints.label=Add XHR breakpoint
+xhrBreakpoints.item.label=URL contains "%S"
```

Comment:
You need to use curly quotes, otherwise tests will fail as soon as you try to land changes in mozilla-central

```
xhrBreakpoints.item.label=URL contains “%S”
```

Also, please add a comment to explain what's going to be the value of that placeholder.